src/main/java/net/sf/jabref/model/entry/BibLatexEntryTypes.java

```diff
@@ -871,4 +871,10 @@ public String getName() {
     public static Optional<EntryType> getType(String name) {
         return ALL.stream().filter(e -> e.getName().equalsIgnoreCase(name)).findFirst();
     }
+
+    public static final List<String> BIBLATEX_PERSON_NAME_FIELDS = Arrays.asList("author", "editor", "editora",
```

Comment:
No, not all field, only some (configurable?). Yes, editora, editorb, editorc are defined in BibLatex. There are also namea, nameb, ... if one would like but these are not general fields in the standard but are defined by certain styles.